src/vs/workbench/contrib/tasks/browser/terminalTaskSystem.ts

```diff
@@ -963,13 +963,6 @@ export class TerminalTaskSystem extends Disposable implements ITaskSystem {
 				for (let i = bufferLines.length - 1; i >= 0; i--) {
 					watchingProblemMatcher.processLine(bufferLines[i]);
 				}
-				if (!delayer) {
```

Comment:
don't think this is needed -  it's used in the other place processLine is called, but that's bc it's within `onData` and thus needs to not process lines all of the time